android/app/src/main/kotlin/coop/polypoly/polypod/FeatureFragment.kt

```diff
@@ -73,6 +74,12 @@ private enum class ActionButton(val action: Action, val buttonId: Int) {
     SEARCH(Action.SEARCH, R.id.search_button)
 }
 
+open class ExternalFile(url: String, name: String, size: Long) {
+    val url: String = url
+    val name: String = name
+    val size: Long = size
```

Comment:
Nit: You can write it a bit shorter in Kotlin, also it could be a data class, something like:

`data class ExternalFile(val url: String, val name: String, val size: Long)`

That syntax creates constructor and declares fields in one go.